presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -14,30 +14,38 @@
 
 package com.facebook.presto.spi.statistics;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import static com.facebook.presto.spi.statistics.Estimate.unknownValue;
+import static java.util.Collections.unmodifiableMap;
 import static java.util.Objects.requireNonNull;
 
 public final class ColumnStatistics
 {
-    public static final ColumnStatistics EMPTY_STATISTICS = ColumnStatistics.builder().build();
-
-    private final Estimate dataSize;
-    private final Estimate nullsCount;
-    private final Estimate distinctValuesCount;
+    private final Map<String, Estimate> statistics;
```

Comment:
How about we keep both?
We can use method interfaces (accessing map) when we are interested in single stat, but we may access whole map if we just wan't to know what is provided.

Also map interface may be useful if it will ever be the case that two systems will use the stats interface to communicate with presto only being used to pass those data. If it will pass interface with map it will be a gate to extending passed data beyond what is understood by presto. I know such solutions aren't perfect, but lets be honest - we've done worst things before (QG, khem, khem) and having such doors may become handy.